testng-core/src/main/java/org/testng/internal/ClonedMethod.java

```diff
@@ -340,6 +340,16 @@ public void setInterceptedPriority(int priority) {
     // ignored
   }
 
+  @Override
+  public int getClassHierarchyPriority() {
+    return m_method.getClassHierarchyPriority();
+  }
+
+  @Override
+  public void setClassHierarchyPriority(int priority) {
+    // ignored
```

Comment:
I have no idea why all "set priorities" are ignored. I feel an exception is a better option.
What do you think?

Reply:
I just tried to follow the same concept as we have for setPriority and setInterceptedPriority.
I can try to add exceptions, but in that case we need to add it for all.